### ISSUE TITLE
fcos nodes fail to start crio service

### ISSUE THREAD

**Issue by gtema** [openshift/okd#51]
# Version

4.3.0-0.okd-2020-01-24-215611 on FedoraCoreOS31

# What happened?

Master nodes fail (FCOS image) to start crio service, due to missing /usr/share/containers/oci/hooks.d dir
```
Jan 24 15:43:31 osinfra-cqr7z-master-2 crio[3282]: time="2020-01-24 15:43:31.186522987Z" level=fatal msg="runtime config: invalid hooks_dir: stat /usr/share/containers/oci/hooks.d: no such file or directory: stat /usr/share/containers/oci/hooks.d: no such file or directory"
```

# What you expected to happen?

/usr/share/containers/oci/hooks.d directory exists in the image or configuration is adapted to remove it

# How to reproduce it (as minimally and precisely as possible)?

install-config.yaml
```
apiVersion: v1
baseDomain: fake.com
compute:
- name: worker
  replicas: 0
controlPlane:
  name: master
  replicas: 3
metadata:
  name: osinfra
networking:
  clusterNetworks:
  - cidr: 10.254.0.0/16
    hostPrefix: 24
  networkType: OpenShiftSDN
  serviceNetwork:
  - 172.30.0.0/16
  machineNetwork:
  - cidr: 192.168.190.0/24
platform:
  none: {}
pullSecret: '{"auths":{"fake":{"auth": "bar"}}}'
sshKey: my_key
```

Follow https://github.com/openshift/installer/blob/fcos/docs/user/openstack/install_upi.md instructions with fcos-31 image up to (including) starting master nodes. Login to any master node after it refetched image.
System reports, that crio.service failed to start

# Anything else we need to know?
After manual editing of the /etc/crio/crio.conf and restarting crio service bootstraping continues.

The same problem exists in worker nodes as well.

After all this leads to the degradation of the machine-configs operator state

**Comment by gtema** [comment]
moved from https://github.com/openshift/installer/issues/2978

**Comment by vrutkovs** [comment]
Duplicate of #42

**Comment by gtema** [comment]
why is it a duplicate?

**Comment by vrutkovs** [comment]
The cause of this is that MCD is not applying necessary crio config before starting it, see https://github.com/openshift/machine-config-operator/pull/1311

**Comment by gtema** [comment]
ok, thks.
